# Capitalize enumerated type constants. (capitalize-constants)

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
// fill me in
```

Examples of **correct** code for this rule:

```js
// fill me in
```

### Options

This rule has an object option:

- `"ignoreCapitalization": true` allows names to be capitalized in any way.
- `"ignoreEntities": true` allows names to use expanded HTML entities, i.e. `&trade;`.
- `"ignoreUnrecognized": true` allows unrecognized value names.
